src/main-mongo.ts

```diff
@@ -0,0 +1,29 @@
+// tslint: disable: no - unused - expression;
```

Comment:
Same thing here. Perhaps in development time you're adding testing scenarios and forgot to remove them all before pushing.